__tests__/index.tests.js

```diff
@@ -224,4 +225,55 @@ describe('error 404 GET /api/topics/', ()=>{
         .get('/api/articles/13/comments')
         .expect(200)
       })
-    })
\ No newline at end of file
+    })
+    describe("Post 201: inserts a new comment to some article", ()=>{ 
+      let dateCreated=""
+      test('POST:201 inserts a new comment to the db and sends the new comment back to the client', () => {
+        const newComment = {
+          username: 'butter_bridge',
+          body: 'I do not believe you'
+        };
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(newComment)
+          .expect(201)
+          .then((response) => {
+            console.log(response.body)
+            expect(response.status).toBe(201); // Check the HTTP status code
+    expect(response.body.comment).toHaveProperty('created_at'); // Check if 'comment' property exists
+    expect(response.body.comment.body).toEqual(newComment.body);
+    expect(response.body.comment.author).toEqual(newComment.username);
+    expect(response.body.comment.votes).toEqual(0)
+    expect(response.body.comment.article_id).toEqual(1)
+     
+     
+      })
+      
+        })
+        test('POST:404 on bad path', ()=>{
+          
```

Comment:
You do not need this test here and a bad path should already have been tested for in a previous ticket.